tpp-app/src/home/pages/CycleHistoryScreen.js

```diff
@@ -67,18 +70,20 @@ export default function CycleHistoryScreen({navigation}){
         .catch(() => setOnPeriod(DEFAULTS.ON_PERIOD))
     }, []));
 
-    //update the cycles being rendered to reflect newly selected year.
+    //get intervals for all stored years
     useFocusEffect(
-        React.useCallback(() => {
-            console.log("on focus for intervalss")
-            CycleService.GETCycleHistoryByYear(selectedYear).then(
-                intervals => {
-                    setCurrentIntervals(intervals);
-                }
-            )
-            .catch(() => setCurrentIntervals(DEFAULTS.CURRENT_INTERVALS)) //error case, render empty card
+        React.useCallback(
```

Comment:
Very tiny note, but you might as well add `useCallback` to the list of React imports so you don't have to prefix React dot.